lib/samples/manage_bookmarks/manage_bookmarks_sample.dart

```diff
@@ -0,0 +1,252 @@
+import 'package:flutter/material.dart';
+
+class ManageBookmarksSample extends StatefulWidget {
+  const ManageBookmarksSample({super.key});
+
+  @override
+  State<ManageBookmarksSample> createState() => _ManageBookmarksSampleState();
+}
+
+class _ManageBookmarksSampleState extends State<ManageBookmarksSample> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // Declare a list of bookmarks.
+  late final List<Bookmark> _bookmarks;
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // A flag for when the bookmarks bottom sheet is visible.
+  var _bookmarksVisible = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    // A button to add a bookmark at the current Viewpoint.
+                    ElevatedButton(
+                      onPressed: addBookmark,
+                      child: const Text('Add Bookmark'),
+                    ),
+                    // A button to list the bookmarks.
+                    ElevatedButton(
+                      onPressed: () => setState(() => _bookmarksVisible = true),
+                      child: const Text('Bookmarks'),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: SizedBox.expand(
+                child: Container(
+                  color: Colors.white30,
+                  child: const Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      // The Bookmarks bottom sheet.
+      bottomSheet: _bookmarksVisible ? buildBookmarks(context) : null,
+    );
+  }
+
+  // The build method for the Bookmarks bottom sheet.
+  Widget buildBookmarks(BuildContext context) {
+    return Container(
+      constraints: BoxConstraints(
+        maxHeight: MediaQuery.sizeOf(context).height * 0.5,
+      ),
+      padding: EdgeInsets.fromLTRB(
+        20.0,
+        0.0,
+        20.0,
+        max(
+          20.0,
+          View.of(context).viewPadding.bottom /
+              View.of(context).devicePixelRatio,
+        ),
+      ),
+      child: SingleChildScrollView(
```

Comment:
The only thing about the scroll being here is that the title and 'x' also scroll. It feels like the list should scroll, but you should always be able to close the sheet if that makes sense?

Reply:
I adjusted the widgets a bit so now the bookmarks scroll by themselves